### PR TITLE
LPS-204851 - Unsupported options appears in UI for changes in an already-published Publication

### PR DESCRIPTION
**Pull request by david-truong**

### CODE REVIEW

**Comment by david-truong** [comment]
ci:test:publications

**Comment by david-truong** [comment]
ci:test:sf

**Comment by david-truong** [comment]
ci:test:publications

**Comment by david-truong** [comment]
ci:test:relevant

File: modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/display/context/ViewChangesDisplayContext.java [hunk 1]
```diff
@@ -214,27 +214,29 @@ public List<FDSActionDropdownItem> getFDSActionDropdownItems() {
...
-				).buildString(),
-				"move-folder", "move-changes",
-				_language.get(_httpServletRequest, "move-changes"), "post",
-				"move-changes", null));
+				"get", null));
 
-		if ((_ctCollection.getStatus() == WorkflowConstants.STATUS_DRAFT) ||
-			(_ctCollection.getStatus() == WorkflowConstants.STATUS_PENDING)) {
+		if ((_ctCollection.getStatus() == WorkflowConstants.STATUS_DRAFT) &&
+			(_ctCollection.getStatus() != WorkflowConstants.STATUS_PENDING)) {
```
Comment (pei-jung) [inline modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/display/context/ViewChangesDisplayContext.java:220]: @david-truong is this condition right? Shouldn't this be 
```
if ((_ctCollection.getStatus() == WorkflowConstants.STATUS_DRAFT) && (_ctCollection.getStatus() == WorkflowConstants.STATUS_PENDING))
```

Comment (david-truong) [reply modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/display/context/ViewChangesDisplayContext.java:220]: thanks for catching that

**Comment by david-truong** [comment]
ci:test:sf

**Comment by david-truong** [comment]
ci:test:publications

**Comment by david-truong** [comment]
ci:test:relevant

**Comment by david-truong** [comment]
@pei-jung do you think this is ready to forward?

**Comment by pei-jung** [comment]
LGTM. Failures look unrelated.

**Comment by pei-jung** [comment]
ci:forward:force

**Comment by pei-jung** [comment]
ci:test:sf

**Comment by pei-jung** [comment]
@david-truong SF failure is related to a recent change in the source formatter https://liferay.slack.com/archives/C1SJ64S9X/p1704806532852469?thread_ts=1704799263.629319&cid=C1SJ64S9X. Do you want me to manually forward?

**Comment by pei-jung** [comment]
Forwarded to BChan https://github.com/brianchandotcom/liferay-portal/pull/146164.

**Pull request closed without merging**